### PR TITLE
Limit result logging, exclude Monitoring tests for broker profile

### PR DESCRIPTION
**Pull request by ahelmbr**
- In situation with heavy load (could be misconfigured) the web console reporting can be flooded with log messages, that could lead into an OOM of the browser. The limit is now shifting older entries away. 
- Specific profiles don't need all internal tests from the internal Monitor, the results from there will be skipped in that case.

### CODE REVIEW

File: tck/src/main/java/org/eclipse/sparkplug/tck/test/TCKTest.java [hunk 1]
```diff
@@ -43,20 +43,26 @@
...
 
-    }
+	protected @NotNull Utils.Profile profile;
 
-    public void onAuthenticationSuccessful(AuthenticationSuccessfulInput authenticationSuccessfulInput) {
+	public void onMqttConnectionStart(ConnectionStartInput connectionStartInput) {
+	}
 
-    }
+	public void onAuthenticationSuccessful(AuthenticationSuccessfulInput authenticationSuccessfulInput) {
```
Comment (icraggs-sparkplug) [inline tck/src/main/java/org/eclipse/sparkplug/tck/test/TCKTest.java:62]: I still need the monitor to be called for the these and other events when the profile is not broker.

And connect, disconnect, subscribe and publish should not call the Monitor if the current profile is broker, I think,

File: tck/src/main/java/org/eclipse/sparkplug/tck/test/common/Utils.java [hunk 2]
```diff
@@ -237,6 +237,12 @@ private enum HostState {
 		OFFLINE
 	}
 
+	public enum Profile {
```
Comment (icraggs-sparkplug) [inline tck/src/main/java/org/eclipse/sparkplug/tck/test/common/Utils.java:240]: It seems to me that enums are constants and would fit better in the Constants class? (Yes it's named TopicConstants at the moment but we could rename it to just Constants. It already contains constants that are not topics).

**Review by icraggs-sparkplug — CHANGES_REQUESTED** [review]

File: tck/src/main/java/org/eclipse/sparkplug/tck/test/common/Utils.java (hunk 2 above)
Comment (icraggs-sparkplug) [reply tck/src/main/java/org/eclipse/sparkplug/tck/test/common/Utils.java:240]: Actually I could do this in another PR.

File: tck/src/main/java/org/eclipse/sparkplug/tck/test/TCKTest.java (hunk 1 above)
Comment (ahelmbr) [reply tck/src/main/java/org/eclipse/sparkplug/tck/test/TCKTest.java:62]: added the condition for the callbacks

File: tck/src/main/java/org/eclipse/sparkplug/tck/test/common/Utils.java (hunk 2 above)
Comment (ahelmbr) [reply tck/src/main/java/org/eclipse/sparkplug/tck/test/common/Utils.java:240]: yes pls. do this in another PR after this one

Comment (icraggs-sparkplug) [reply tck/src/main/java/org/eclipse/sparkplug/tck/test/common/Utils.java:240]: will do

**Review by icraggs-sparkplug — APPROVED** [review]

**Pull request merged**